specification/relay/resource-manager/Microsoft.Relay/stable/2016-07-01/relay.json

```diff
@@ -403,6 +403,47 @@
         "x-ms-pageable": {
           "nextLinkName": "nextLink"
         }
+      },
+      "post": {
```

Comment:
Why is there a POST to list all authorization rules? Is it because you need to expose some keys.

Reply:
it list all Authorizationrules and no keys are exposed.

Reply:
Yes. In this API version, POST on the AuthorizationRules URL functions like a GET but does return the keys associated with the Rule.  Whereas GET doesn't return keys.  In future API versions 2015 we use the RegenerateKeys, ListKeys actions to get and manipulate keys to align better with other RPs.